src/main/java/org/wildfly/swarm/plugin/process/ModuleFiller.java

```diff
@@ -69,16 +73,23 @@
 import org.jboss.shrinkwrap.descriptor.spi.node.NodeImporter;
 import org.jboss.shrinkwrap.descriptor.spi.node.dom.XmlDomNodeImporterImpl;
 import org.wildfly.swarm.plugin.FractionMetadata;
+import org.wildfly.swarm.plugin.utils.FilteringHashSet;
+import org.wildfly.swarm.plugin.utils.HackedModuleDescriptor;
 
 /**
  * @author Bob McWhirter
  * @author Ken Finnigan
  * @author <a href="mailto:jperkins@redhat.com">James R. Perkins</a>
  */
-public class ModuleFiller implements Function<FractionMetadata, FractionMetadata> {
+public class ModuleFiller {
 
     private final Log log;
 
+    private final Predicate<String> noPlatformModules = module -> !module.startsWith("java.")
+            && !module.startsWith("javafx.")
+            && !module.startsWith("jdk.")
+            && !module.startsWith("org.jboss.modules");
```

Comment:
Because there were no platform modules previously :-) JBoss Modules is JPMS-ready (well, sort of, I don't really know) and emulates the platform modules on Java 8. In any case (be it Java 8 or Java 9+), there is no `module.xml` for these modules that could be analyzed.